CMP1903M Assessment 1 Base Code/Analyse.cs

```diff
@@ -24,12 +24,38 @@ public List<int> analyseText(string input)
             //5. Number of lower case letters
             List<int> values = new List<int>();
             //Initialise all the values in the list to '0'
+            var vowels = new HashSet<char> {'a','e','i','o','u'};
             for(int i = 0; i<5; i++)
             {
                 values.Add(0);
             }
 
-
+            foreach (string s in input)
+            {
+                values[0]++;
+                for (int i = 0; i < s.Length; i++)
+                {
+                    if (Char.IsLetter(s[i]))
+                    {                    
+                        if (Char.IsUpper(s[i]))
+                        {
+                            values[3]++;
+                        }
+                        else
+                        {
+                            values[4]++;
+                        }
+                        if (vowels.Contains(Char.ToLower(s[i])))
+                        {
+                            values[1]++;
+                        }
+                        else
+                        {
+                            values[2]++;
+                        }
+                    }
```

Comment:
Would maybe make sense to have the sentence counter in with the other counters? Could add a letter frequency checker too.